GateWatcher/aioniq/_meta/smart-descriptions.json

```
@@ -0,0 +1,91 @@
+[{
+    "value": "Infected payload {file.name} hosted on {url.domain} was blocked: {gatewatcher.malcore.detail_threat_found}",
+    "conditions": [
+        {
+            "field": "gatewatcher.type",
+            "value": "malcore"
+        },
+        {
+            "field": "state",
+            "value": "Infected"
+        },
+        {
+            "field": "url.domain"
+        },
+        {
+            "field": "file.name"
+        },
+        {
+            "field": "source.address"
+        },
+        {
+            "field": "gatewatcher.malcore.detail_threat_found"
+        }
+    ],
+    "relationships": [
+        {
+            "source": "source.address",
+            "target": "url.domain",
+            "type": "connection was denied"
+        },
+        {
+            "source": "url.domain",
+            "target": "file.name",
+            "type": "host infected payload"
+        }
+    ]
+},{
+    "value": "Alert {rule.name} raised with severity {event.severity}",
+    "conditions": [
+        {
+            "field": "event.kind",
+            "value": "alert"
+        },
+        {
+            "field": "event.severity"
+        },
+        {
+            "field": "rule.name"
+        }
+    ]
+},{
+    "value": "{file.name} from {url.domain} analyzed",
+    "conditions": [
+        {
+            "field": "gatewatcher.event_type",
+            "value": "fileinfo"
+        },
+        {
+            "field": "url.domain"
+        },
+        {
+            "field": "file.name"
+        }
+    ]
+},{
+    "value": "Connection from {source.address} to {url.domain}",
```

```suggestion
    "value": "{gatewatcher.event_type} connection from {source.address} to {url.domain}",
```

Improvement suggestion :smiley: 

It's always going to be `http` but the user can click on it to filter/search this value

---

Fixed